packages/proposal-sign-extension-ops/test/wast/empty/output.wast

```diff
@@ -0,0 +1,4 @@
+(module
+  (func)
```

Comment:
I'm not sure this test is very useful

Reply:
Ah this was my test to check that the test setup works before having the transform. I dont think it is useless though, empty function body might trigger an edge case somewhere. But we might also remove it